### ISSUE TITLE
dist/offline_installer/redhat: HEAD scylla_setup does not accepts an argument passed from offline installer

### ISSUE THREAD

**Issue by syuu1228** [scylladb/scylla#3995]
```
Complete!
usage: scylla_setup [-h] (--disks DISKS | --no-raid-setup) [--nic NIC]
                    [--ntp-domain NTP_DOMAIN] [--ami] [--setup-nic-and-disks]
                    [--developer-mode] [--no-ec2-check] [--no-kernel-check]
                    [--no-verify-package] [--no-enable-service]
                    [--no-selinux-setup] [--no-bootparam-setup]
                    [--no-ntp-setup] [--no-coredump-setup]
                    [--no-sysconfig-setup] [--no-io-setup]
                    [--no-version-check] [--no-node-exporter]
                    [--no-cpuscaling-setup] [--no-fstrim-setup]
scylla_setup: error: one of the arguments --disks --no-raid-setup is required
```

**Comment by slivne** [comment]
@glommer is this fixed ?

**Comment by slivne** [comment]
@syuu1228 / @penberg is this fixed ?

**Comment by syuu1228** [comment]
@slivne It was https://github.com/scylladb/scylla/issues/4065 and fixed by https://github.com/scylladb/scylla/commit/54bc0ce70dd87735108e59ad81e4e4cf6ac9af4d

**Comment by avikivity** [comment]
Fixed by 54bc0ce70dd87735108e59ad81e4e4cf6ac9af4d
